coding-standards-and-naming.rst

```diff
@@ -2,155 +2,20 @@
  Coding Standards and Naming Policy
 ####################################
 
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in `RFC 2119
+<https://datatracker.ietf.org/doc/html/rfc2119>`_.
+
 *********
- Classes
+ Methods
```

Comment:
Given that methods have the same acronym requirement as classes, but does not explain it, should this be moved down?

Reply:
The section on acronyms is also inconsistent, as it writes "Abbreviations and acronyms as well as initialisms SHOULD be avoided wherever possible, unless they are much more widely used than the long form (e.g. HTTP or
URL)." — that is no longer the case, they should not be HTTP or URL, but Http and Url (As the examples indicate).